client/my-sites/plans/main.jsx

```diff
@@ -473,33 +476,40 @@ class Plans extends Component {
-	const selectedSiteId = getSelectedSiteId( state );
-	const currentPlan = getCurrentPlan( state, selectedSiteId );
-	const currentPlanIntervalType = getIntervalTypeForTerm(
-		getPlan( currentPlan?.productSlug )?.term
-	);
+const ConnectedPlans = connect(
+	( state ) => {
+		const selectedSiteId = getSelectedSiteId( state );
+		const currentPlan = getCurrentPlan( state, selectedSiteId );
+		const currentPlanIntervalType = getIntervalTypeForTerm(
+			getPlan( currentPlan?.productSlug )?.term
+		);
 
-	return {
-		currentPlan,
-		currentPlanIntervalType,
-		purchase: currentPlan ? getByPurchaseId( state, currentPlan.id ) : null,
-		selectedSite: getSelectedSite( state ),
-		canAccessPlans: canCurrentUser( state, getSelectedSiteId( state ), 'manage_options' ),
-		isWPForTeamsSite: isSiteWPForTeams( state, selectedSiteId ),
-		isSiteEligibleForMonthlyPlan: isEligibleForWpComMonthlyPlan( state, selectedSiteId ),
-		plansLoaded: Boolean( getPlanSlug( state, getPlan( PLAN_FREE )?.getProductId() || 0 ) ),
-		isDomainAndPlanPackageFlow: !! getCurrentQueryArguments( state )?.domainAndPlanPackage,
-		isJetpackNotAtomic: isJetpackSite( state, selectedSiteId, { treatAtomicAsJetpackSite: false } ),
-		isDomainUpsell:
-			!! getCurrentQueryArguments( state )?.domainAndPlanPackage &&
-			!! getCurrentQueryArguments( state )?.domain,
-		isDomainUpsellSuggested: getCurrentQueryArguments( state )?.domain === 'true',
-		isFreePlan: isFreePlanProduct( currentPlan ),
-		domainFromHomeUpsellFlow: getDomainFromHomeUpsellInQuery( state ),
-		siteHasLegacyStorage: siteHasFeature( state, selectedSiteId, FEATURE_LEGACY_STORAGE_200GB ),
-	};
-} )( withCartKey( withShoppingCart( localize( Plans ) ) ) );
+		return {
+			currentPlan,
+			currentPlanIntervalType,
+			purchase: currentPlan ? getByPurchaseId( state, currentPlan.id ) : null,
+			selectedSite: getSelectedSite( state ),
+			canAccessPlans: canCurrentUser( state, getSelectedSiteId( state ), 'manage_options' ),
+			isWPForTeamsSite: isSiteWPForTeams( state, selectedSiteId ),
+			isSiteEligibleForMonthlyPlan: isEligibleForWpComMonthlyPlan( state, selectedSiteId ),
+			plansLoaded: Boolean( getPlanSlug( state, getPlan( PLAN_FREE )?.getProductId() || 0 ) ),
+			isDomainAndPlanPackageFlow: !! getCurrentQueryArguments( state )?.domainAndPlanPackage,
+			isJetpackNotAtomic: isJetpackSite( state, selectedSiteId, {
+				treatAtomicAsJetpackSite: false,
+			} ),
+			isDomainUpsell:
+				!! getCurrentQueryArguments( state )?.domainAndPlanPackage &&
+				!! getCurrentQueryArguments( state )?.domain,
+			isDomainUpsellSuggested: getCurrentQueryArguments( state )?.domain === 'true',
+			isFreePlan: isFreePlanProduct( currentPlan ),
+			domainFromHomeUpsellFlow: getDomainFromHomeUpsellInQuery( state ),
+			siteHasLegacyStorage: siteHasFeature( state, selectedSiteId, FEATURE_LEGACY_STORAGE_200GB ),
+		};
+	},
+	( dispatch ) => ( {
+		fetchSitePlans: ( siteId ) => dispatch( fetchSitePlans( siteId ) ),
+	} )
```

Comment:
@candy02058912 Would you mind adding Github comments about what's changing here, to make review easier? It looks like a bunch of lines got included in the diff. 😅

Reply:
Most diffs are easier to read without whitespace (`git diff -w`) https://github.com/Automattic/wp-calypso/pull/87595/files?w=1

You can toggle it off in the UI but it doesn't save between PR's, I use the a firefox addon to force it by default but it looks like chrome has a similar addon too:

https://addons.mozilla.org/en-US/firefox/addon/github-whitespace-disabler/
https://chromewebstore.google.com/detail/github-whitespace/fnpkdafamnbjoldglihkjjdicofghccm?pli=1

Reply:
TIL thanks for sharing the trick!

Reply:
We can just hide the whitespace from settings

![image](https://github.com/Automattic/wp-calypso/assets/13596067/ce6c7dfc-6d56-4371-822f-91d33adc3d35)

Reply:
TIL also, thanks for the tip!

Reply:
Do they save the setting between PRs now?